<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SpecialDialog</class>
 <widget class="QDialog" name="SpecialDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>740</width>
    <height>489</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>50</y>
     <width>291</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>26</pointsize>
     <italic>true</italic>
     <underline>true</underline>
    </font>
   </property>
   <property name="text">
    <string>最近的特价商品！！！</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>100</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>请在前面打钩就可以了</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>420</y>
     <width>731</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="clearPushButton">
      <property name="text">
       <string>清除</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="returnPushButton">
      <property name="text">
       <string>返回</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="payPushButton">
      <property name="text">
       <string>确定</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_4">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QCheckBox" name="littercheckBox">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>310</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>小苹果</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>120</y>
     <width>181</width>
     <height>251</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="picture.qrc">:/new/prefix0/resource/icecream.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QSpinBox" name="kiviSpinBox">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>140</y>
     <width>141</width>
     <height>22</height>
    </rect>
   </property>
   <property name="suffix">
    <string>/斤</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="yeporkSpinBox">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>230</y>
     <width>141</width>
     <height>22</height>
    </rect>
   </property>
   <property name="suffix">
    <string>/斤</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="crawishSpinBox">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>310</y>
     <width>141</width>
     <height>22</height>
    </rect>
   </property>
   <property name="suffix">
    <string>/斤</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="legsSpinBox">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>150</y>
     <width>141</width>
     <height>22</height>
    </rect>
   </property>
   <property name="suffix">
    <string>/斤</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="oriangesSpinBox">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>230</y>
     <width>141</width>
     <height>22</height>
    </rect>
   </property>
   <property name="suffix">
    <string>/斤</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="litterSpinBox">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>310</y>
     <width>141</width>
     <height>22</height>
    </rect>
   </property>
   <property name="suffix">
    <string>/斤</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="legsCheckBox">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>150</y>
     <width>59</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>鸡腿</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="kiviCheckBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>59</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>猕猴桃</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="oriangesCheckBox">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>230</y>
     <width>59</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>水晶橙</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="yeporkCheckBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>230</y>
     <width>59</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>野猪肉</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="crawishCheckBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>310</y>
     <width>59</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>小龙虾</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="picture.qrc"/>
 </resources>
 <connections/>
</ui>
